crates/rpc-types-eth/src/filter.rs

```diff
@@ -800,135 +1013,6 @@ where
     }
 }
 
-/// Support for matching [Filter]s
-#[derive(Debug, Default)]
-pub struct FilteredParams {
-    /// The original filter, if any
-    pub filter: Option<Filter>,
-}
```

Comment:
what's the right way to handle an optional filter now?

Reply:
an empty Filter via Filter::default

Reply:
another easy approach would be `filter_opt.map(|f| f.matches(thing)).unwrap_or(true)`